openupgradelib/openupgrade.py

```diff
@@ -1079,6 +1079,12 @@ def rename_models(cr, model_spec):
                     "UPDATE mail_activity SET res_model=%s where res_model=%s",
                     (new, old),
                 )
+        if is_module_installed(cr, "rating"):
```

Comment:
Better check if table existe.

Reply:
@legalsylvain ok as you wish sir

Reply:
@pedrobaeza Okay, done sir, i saw the above check both so i copy :v

Reply:
Yes, it's true that the module installed check is used up on the code for `mail`. I suppose that one was for not having to check a lot of table existence and as the module name is expected to be constant. In this case, I think one check is enough.

Reply:
Okay, that 's fine by me